images/cassandra/config/template.apko.yaml

```diff
@@ -18,12 +18,12 @@ accounts:
 work-dir: /
 
 entrypoint:
-  command: /opt/cassandra/cassandra/bin/cassandra -f
+  command: cassandra -f
```

Comment:
I think we may still want to have this be an absolute path.

Reply:
Addressed in the latest commit